tests/treeTest.py

```diff
@@ -1,18 +1,15 @@
+import os
 import unittest
-
```

Comment:
It's not big deal, but generally it looks nicer when you separate imports into blocks. If you really want to remove this line, make sure that you remove in all files, and make it in separate commit, as it only relates to code style.